src/main/java/io/github/thebusybiscuit/slimefun4/core/attributes/TierAccessible.java

```diff
@@ -0,0 +1,22 @@
+package io.github.thebusybiscuit.slimefun4.core.attributes;
+
+import io.github.thebusybiscuit.slimefun4.core.accessibility.AccessData;
+import io.github.thebusybiscuit.slimefun4.core.accessibility.AccessManager;
+
+import javax.annotation.Nonnull;
+
+
+/**
+ * Represents anything with tiered access.
+ *
+ * @see io.github.thebusybiscuit.slimefun4.core.accessibility.AccessLevel
+ * @see AccessData
+ *
+ * @author md5sha256
+ */
+public interface TierAccessible {
```

Comment:
Single-Method-Interfaces should be marked as `@FunctionalInterface`, unless you got explicit plans to add more methods.

Reply:
Noted, I'll add the annotation if I do not add more methods to the interface.

Reply:
Great :+1: